src/components/common/settings/sidebar.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import {
+  Banknote,
+  Bell,
+  ChevronLeft,
+  Database,
+  Globe,
+  LucideIcon,
+  User,
+  UserPlus,
+} from "lucide-react";
+import Link from "next/link";
+import { usePathname, useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+
+import { cn } from "~/lib/utils";
+
+type SettingsSidebarProperties = {
+  id?: number;
+  label: string;
+  icon: LucideIcon;
+  link: string;
+};
+
+const Settings_Sidebar_Links: SettingsSidebarProperties[] = [
```

Comment:
Is this pascal-kebab case? Either use camelCase `settingsSidebarLinks` or use `SETTINGS_SIDEBAR_LINKS` since it looks like a constant